README.md

```diff
@@ -1,2 +1,51 @@
 # lint
+
 Go code linters for Project Contour.
+
+## running
+```bash
+go install .
+go vet -vettool (which lint) ./...
+```
+
```

Comment:
I couldn't figure out how to run this with `go fix`, so here or in a different issue we should give an example :)

Reply:
I can't remember what was the command :D can we add this later?